File: task_runner_service/backend/src/services/tool_runtime/mcp_server/support.rs
// SPDX-License-Identifier: PolyForm-Noncommercial-1.0.0
// Required Notice: Copyright (c) 2025 AI Chat Team

#[path = "support/access.rs"]
mod access;
#[path = "support/graph.rs"]
mod graph;
#[path = "support/schema.rs"]
mod schema;

pub(super) use access::*;
pub(super) use graph::*;
pub(super) use schema::*;

File: admin_console/src/modules/plugin-management/pages/agentPrompts/support.ts
// SPDX-License-Identifier: PolyForm-Noncommercial-1.0.0
// Required Notice: Copyright (c) 2025 AI Chat Team

import type { AgentPromptVendor } from '../../types';

export const AGENT_PROMPT_VENDORS: AgentPromptVendor[] = ['glm', 'deepseek', 'gpt', 'kimi'];

export function agentPromptVendorLabel(vendor: AgentPromptVendor): string {
  return {
    glm: 'GLM',
    deepseek: 'DeepSeek',
    gpt: 'GPT / OpenAI',
    kimi: 'Kimi / Moonshot',
  }[vendor];
}
